bubblewrap.c

```diff
@@ -3214,6 +3279,11 @@ main (int    argc,
 
   /* Should be the last thing before execve() so that filters don't
    * need to handle anything above */
+  if (opt_landlock_ruleset_fd>-1)
+    {
+      if (landlock_restrict_self(opt_landlock_ruleset_fd,0)!=0)
+        warn("An error has occured while enabling Landlock restrictions. The sandbox will start without Landlock protection.\n");
```

Comment:
If we want Landlock to be a security restriction, then it should fail closed. We can have a separate `--landlock-try` option for "try to do Landlock, but fall back to not", if you want to also support it as an optional hardening thing - but the default should be fail-closed.